src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,17 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
+
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User[] getUsers = userService.getUsers();
+        for (User user : getUsers) {
+            if (user.getEmail() == email && user.getPassword() == password) {
+                return true;
+            }
+        }
```

Comment:
I would recommend to make use of `UserService` when implementing login logic here. Also, consider making it class member, rather than local variable.